sched/queue/memory/simple.go

```diff
@@ -0,0 +1,81 @@
+package memory
+
+// In-memory Scheduler Queue
+
+import (
+	"fmt"
+	"github.com/scootdev/scoot/sched"
+	"github.com/scootdev/scoot/sched/queue"
+	"time"
+)
+
+func NewSimpleQueue() (queue.Queue, chan sched.Job) {
```

Comment:
TBH, I find this whole thing confusing. Do we need all these channels? Also, I think we'll need a peek() method so we don't drop jobs, and more detail about dequeu would be helpful.